libportal/meson.build

```diff
@@ -138,7 +148,19 @@ if 'gtk3' in backends
       link_with: [libportal_gtk3],
       includes: [libportal_gir[0], 'Gtk-3.0'],
       install: true,
+      extra_args: ['--c-include=libportal-gtk3/portal-gtk3.h'],
+      export_packages: ['libportal-gtk3'],
      )
+
+    if vapi
+      libportal_gtk3_vapi = gnome.generate_vapi('libportal-gtk3',
```

Comment:
This variable is not used anywhere, I don't think you need it. (Same for the GTK4 variable)

Reply:
If any Vala project wants to include libportal as a subproject, it will need this variable, otherwise valac will not be able to find the corresponding vapi

Reply:
I added `libportal_vapi` just in case.